### ISSUE TITLE
add some tags and release?

### ISSUE THREAD

**Issue by wqh17101** [IDEA-Research/Grounded-Segment-Anything#330]

**Comment by rentainhe** [comment]
> _No description provided._

Nice advice, we will collect our updates and add a new tag for Grounded-SAM

**Comment by wqh17101** [comment]
Yeah. Maybe you can release a tag for the current branch as base,

**Comment by Airflyctl** [comment]
> > _No description provided._
> 
> Nice advice, we will collect our updates and add a new tag for Grounded-SAM

So When can we get a release version? Any plans to reach the idea?
